### PR TITLE
Add `--include-original` flag

### LINKED ISSUE #3: Option for also showing original size
```shell
$> gzip-size file.js --original
20 kB, 2 kB gzipped
```
```shell
$> gzip-size file.js --incl-org
20 kB / 2 kB
```

Google ranks on both metrics, so it is indeed useful information.

### PR DESCRIPTION
**Pull request by vltansky**
closes #3

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -18,14 +18,17 @@ $ gzip-size --help
     $ cat <file> | gzip-size
 
   Options
-    --level  Compression level [0-9] (Default: 9)
-    --raw    Display value in bytes
+    --level             Compression level [0-9] (Default: 9)
+    --raw               Display value in bytes
+	  --include-original  Display original size
```
Comment (sindresorhus) [inline readme.md]: Incorrect indentation

File: cli.js [hunk 2]
```diff
@@ -4,28 +4,35 @@ const fs = require('fs');
...
+	  --raw               Display value in bytes
+	  --include-original  Display original size
 
 	Examples
 	  $ gzip-size unicorn.png
 	  192 kB
 	  $ gzip-size unicorn.png --raw
 	  192256
+	  $ gzip-size unicorn.png --include-original
+	  20 kB → 2 kB
```
Comment (sindresorhus) [inline cli.js]: For consistency, the gzip size should be the same as in the other examples.

File: cli.js [hunk 3]
```diff
@@ -4,28 +4,35 @@ const fs = require('fs');
...
 	Usage
 	  $ gzip-size <file>
 	  $ cat <file> | gzip-size
 
 	Options
-	  --level  Compression level [0-9] (Default: 9)
-	  --raw    Display value in bytes
+	  --level             Compression level [0-9] (Default: 9)
+	  --raw               Display value in bytes
+	  --include-original  Display original size
```
Comment (sindresorhus) [inline cli.js]: The description makes it sound like it *only* displays the original size.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: cli.js (hunk 3 above)
Comment (vltansky) [reply cli.js]: Is "Include original size" would be good enough or would you want to suggest any other?

File: cli.js (hunk 2 above)
Comment (vltansky) [reply cli.js]: `1920 kB → 192 kB`?

File: cli.js [hunk 4]
```diff
@@ -4,28 +4,35 @@ const fs = require('fs');
...
 
 	Examples
 	  $ gzip-size unicorn.png
 	  192 kB
 	  $ gzip-size unicorn.png --raw
 	  192256
+	  $ gzip-size unicorn.png --include-original
+	  1920 kB → 192 kB
```
Comment (sindresorhus) [inline cli.js]: This is pretty unrealistic. gzip would not compress 10x for an image.

File: cli.js (hunk 3 above)
Comment (sindresorhus) [reply cli.js]: Yeah, that's fine.

File: cli.js (hunk 4 above)
Comment (vltansky) [reply cli.js]: Guess `392 kB → 192 kB` would be fine, right?

File: cli.js (hunk 3 above)
Comment (vltansky) [reply cli.js]: done

**Pull request merged**